### DISCUSSION TITLE
First Temporal Update - GroupNotFoundError

### DISCUSSION THREAD

**Discussion by Bilalgp** [miniscope/minian#190] [Q&A]
I am running a dataset of TIFF files on the pipeline notebook. Everything works up until I reach the second cell in the subsection of "first temporal update, parameter exploration" and I am not sure why it is present. 

![image](https://user-images.githubusercontent.com/26805330/146448374-5c981765-c65d-4bcb-b4e3-d4981afb2370.png)

![image](https://user-images.githubusercontent.com/26805330/146448319-9306aff0-e954-47ec-8e10-bbd261601bbf.png)

**Comment by phildong** [comment]
it seems like some intermediate results were not found under the intermediate path. Did you clear the path somehow between the run? Is there any saving-related error?

**Comment by hardawayja** [comment]
Hi Phil,

We've also had this error. Is it possible if you have a smaller number of cells that they were filtered out in earlier steps and nothing was created in the intermediate results folder?

thank you very much.

**Comment by phildong** [comment]
Hey @hardawayja that's possible. Although I'd say it's probably not from earlier steps per-se, but it's more like some sort of "empty" folder already exists in the intermediate folder even before this run of minian. Clearing the intermediate folder before running minian usually fix this

**Comment by hardawayja** [comment]
Hey @phildong Thanks we'll try that! Appreciate the help.
